greenplumpython/order.py

```diff
@@ -101,14 +111,15 @@ def __getitem__(self, rows: slice) -> "DataFrame":
     @property
     def dataframe(self) -> "DataFrame":
         """
-        Returns GreenplumPython :class:`~dataframe.DataFrame` associated for ORDER BY
+        Dataframe property.
 
         Returns:
-            GreenplumPython DataFrame
+            :class:`~dataframe.DataFrame` associated with ORDER BY
```

Comment:
This is not the ordered dataframe, it is the original one.
Actually I think we can delete this property, just use `self._dataframe`.

@xuebinsu @Quasimewdo

Reply:
Agree